app/_how-tos/custom-deck-object-defaults.md

```diff
@@ -1,120 +0,0 @@
----
```

Comment:
I went back and forth on removing this. The reference has more information, and I think we're missing additional validations to ensure the default is used.

I'd like to add this back in the future, but for now I think the reference is better